pkg/operation/botanist/component/kubeapiserver/kube_apiserver.go

```diff
@@ -434,13 +448,22 @@ func (k *kubeAPIServer) Deploy(ctx context.Context) error {
 }
 
 func (k *kubeAPIServer) Destroy(ctx context.Context) error {
+	seedK8sVersionGreatEqual121, err := version.CompareVersions(k.values.SeedVersion.String(), ">=", "1.21")
+	if err != nil {
+		return err
+	}
+	seedK8sVersionGreatEqual123, err := version.CompareVersions(k.values.SeedVersion.String(), ">=", "1.23")
+	if err != nil {
+		return err
+	}
```

Comment:
Instead of duplicating this code, you could store the results of `seedK8sVersionGreaterEqual12{1,3}` in the values in the `New` function?

Reply:
Is it okay if we introduce such variables in the `Values` struct?

Reply:
You can make them "private"/lower-case/internal, this should be fine.

Reply:
I still feel this looks odd and not consistent with other components. Can we keep it like this?